sources/react/template-all-icons.js

```diff
@@ -0,0 +1,55 @@
+const toString = array => array.reduce( ( acc, item ) => ( acc = acc + "'" + item + "', " ), '' );
+
+const prepareAllIcons = ( {
+	components,
+	iconsThatNeedOffset,
+	iconsThatNeedOffsetX,
+	iconsThatNeedOffsetY,
+} ) => `
+/** @ssr-ready **/
```

Comment:
The `/** @ssr-ready **/` pragma is obsolete -- let's remove it while we're here (and add a note to the `Changelog` 😄)

Reply:
Done at d5963b9